### DISCUSSION TITLE
When is 1.18 release?

### DISCUSSION THREAD

**Discussion by kulak** [FreshRSS/FreshRSS#3465] [General]
I am on Arch Linux and my current installation is broken due to newer PHP version.  I see that it is fixed in master, but it has not been released yet.  I could go through trouble of building it all myself, but I'd rather stick to AUR updates system.

**Comment by aledeg** [comment]
Soon I guess. There is some pending issues that need to be addressed beforehand.
Of course, help is welcomed.
@Alkarex can give you a better approximation than mine though.

**Reply by Alkarex** [reply]
I hope to have a RC during the week, after the top ~4 issues of https://github.com/FreshRSS/FreshRSS/milestone/46 have been fixed

**Comment by Alkarex** [comment]
We have a release candidate 1.18.0-rc at last :-) Please give it a spin https://github.com/FreshRSS/FreshRSS/issues/3512

**Reply by Frenzie** [reply]
Nothing obviously wrong yet. ;-)

**Comment by Alkarex** [comment]
Done https://github.com/FreshRSS/FreshRSS/releases/tag/1.18.0

**Reply by kulak** [reply]
I just installed it on Manjaro Linux with updated package and it works great!  My machine is on PHP v 8.0.2.  

I am sorry, original question described OS as Arch, which is I use just not in this case.
